client/my-sites/controller.js

```diff
@@ -112,46 +112,62 @@ function renderNoVisibleSites( context ) {
 	);
 }
 
-function isPathAllowedForDomainOnlySite( pathname, domainName ) {
+function renderSelectedSiteIsDomainOnly( { reactContext, selectedSite } ) {
+	const EmptyContentComponent = require( 'components/empty-content' );
+	const { store: reduxStore } = reactContext;
+
+	removeSidebar( reactContext );
+
+	renderWithReduxStore(
+		React.createElement( EmptyContentComponent, {
+			title: i18n.translate( 'This feature is not available for domains' ),
```

Comment:
Alternate string suggestion: 
* `translate( 'This domain is not available.' )`&mdash; translations: **18**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/domain-only-site-featuregate)